components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -168,6 +168,13 @@ export default function ProjectSettingsView() {
                 />
                 {enablePrebuilds && (
```

Comment:
**issue(non-blocking)**: I got the following error on a GitHub repository, not sure if it's related to the changes here.
> GitHub API Error. Status: 422

Reply:
Don't think this is related. It might be a glitch with GitHub. 

> HTTP 422: Unprocessable Entity

Reply:
Not sure if this related to a provider or simply the first repository added.
Setting worked ok in the next projects added.

Cc @svenefftinge because https://github.com/gitpod-io/gitpod/pull/18716#issuecomment-1723525227.